include/mqtt/endpoint.hpp

```diff
@@ -8278,1217 +8325,3808 @@ class endpoint : public std::enable_shared_from_this<endpoint<Socket, Mutex, Loc
+                    );
+                    process_property_id(
+                        std::move(func),
+                        std::move(buf),
+                        rest,
+                        std::move(props),
+                        std::move(handler),
+                        std::move(self)
+                    );
+                },
+                std::move(self)
+            );
+        } break;
+        case v5::property::id::subscription_identifier_available: {
+            std::size_t const len = 1;
+            if (property_length_rest < len) {
+                call_message_size_error_handlers(func);
+                return;
+            }
+            process_nbytes(
+                std::move(func),
+                std::move(buf),
+                len,
+                [
+                    this,
+                    props = std::move(props),
+                    handler = std::move(handler),
+                    rest = property_length_rest - len
+                ]
+                (buffer body, buffer buf, async_handler_t func, this_type_sp self) mutable {
+                    props.emplace_back(
+                        v5::property::subscription_identifier_available(body.begin(), body.end())
+                    );
+                    process_property_id(
+                        std::move(func),
+                        std::move(buf),
+                        rest,
+                        std::move(props),
+                        std::move(handler),
+                        std::move(self)
+                    );
+                },
+                std::move(self)
+            );
+        } break;
+        case v5::property::id::shared_subscription_available: {
+            std::size_t const len = 1;
+            if (property_length_rest < len) {
+                call_message_size_error_handlers(func);
+                return;
+            }
+            process_nbytes(
+                std::move(func),
+                std::move(buf),
+                len,
+                [
+                    this,
+                    props = std::move(props),
+                    handler = std::move(handler),
+                    rest = property_length_rest - len
+                ]
+                (buffer body, buffer buf, async_handler_t func, this_type_sp self) mutable {
+                    props.emplace_back(
+                        v5::property::shared_subscription_available(body.begin(), body.end())
+                    );
+                    process_property_id(
+                        std::move(func),
+                        std::move(buf),
+                        rest,
+                        std::move(props),
+                        std::move(handler),
+                        std::move(self)
+                    );
+                },
+                std::move(self)
+            );
+        } break;
+        }
+    }
+
+    // process common
+
+    templat
```

Comment:
Not using capture variables.

Reply:
I think that the code reference is something wrong. Could you update it?